diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/identifiers/LocalVariablesRule.kt

```diff
@@ -69,7 +69,7 @@ class LocalVariablesRule(private val configRules: List<RulesConfig>) : Rule("loc
     private fun collectLocalPropertiesWithUsages(node: ASTNode) = node
         .findAllVariablesWithUsages { propertyNode ->
             propertyNode.isLocal && propertyNode.name != null && propertyNode.parent is KtBlockExpression &&
-                    (propertyNode.isVar && propertyNode.initializer == null ||
+                    (propertyNode.isVar ||
```

Comment:
This actually was intended as a feature: we collect either unitialized vars or vars, that have one of the specific initializers -  constants or well-known methods. There may be vars that expect side effects from the lines after them and so on. If we want to keep this behavior, what are our options to solve the bug?

Reply:
I guess we can skip such properties in offset.